Description: Fixes test that assumes current year is 2020
 The test outputs the year of a timestamp, and expects the result to be 2020
 Will be fixed upstream, so just bump the expected year to 2021.
Author: Lars Tangvald <lars.tangvald@oracle.com>
Origin: upstream
Bug: https://bugs.mysql.com/bug.php?id=102243
Forwarded: not-needed
Last-Update: 2021-01-13

--- mysql-8.0-8.0.22.orig/mysql-test/r/cast.result
+++ mysql-8.0-8.0.22/mysql-test/r/cast.result
@@ -1757,10 +1757,10 @@ CAST(TIMESTAMP'2010-01-01 08:09:10' AS Y
 2010
 SELECT CAST(TIME'08:09:10' AS YEAR);
 CAST(TIME'08:09:10' AS YEAR)
-2020
+2021
 SELECT CAST(TIME'00:00:00' AS YEAR);
 CAST(TIME'00:00:00' AS YEAR)
-2020
+2021
 SELECT CAST(ST_PointFromText('POINT(10 10)') AS YEAR);
 ERROR HY000: Incorrect arguments to cast_as_year
 CREATE TABLE t AS SELECT CAST("2010" AS YEAR);
